### ISSUE TITLE
Using the trained model for text which does not have a ready made abstract

### ISSUE THREAD

**Issue by learningneo** [abisee/pointer-generator#71]
Hi @abisee, thanks for the wonderful model. Given all the improvements, I did not get any issues in getting the training data and shaping it up for training the model from scratch. The resultant model works decently well for the new news articles which have abstracts. Coming to my question - which may be very ignorant, so apologies in that case in advance - Can I use this model in cases where I do not have abstracts? I want to simply pass the article in the desired format with, say, "  " in abstract fields  and run the model in decode mode - will the model still create the summaries? Or will it straight away give errors? My impression is that since I am not running it in training mode, it should not "need" the abstract for the articles - am I correct in assuming that? In a broader picture, can I use this model in a different context then that of news articles with abstracts and if so, what's the best way to do it?

Thanks a ton!

**Comment by learningneo** [comment]
Hi, I have figured out some of my questions:

- Can I use this model in cases where I do not have abstracts? - Yes. Just used spaces in place of abstracts where I had none and hit the model to create the abstracts and does not give errors. But I am yet to train using coverage so summary created is repetitive.

-  In a broader picture, can I use this model in a different context then that of news articles with abstracts and if so, what's the best way to do it? - I tried using Emails and results are, as expected, very naive. Need to work on a better way for creating an abstract for conversational.
